src/recommendationservice/Dockerfile.local.dev

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
```

Comment:
I'd suggest we don't include any `*.local` files, from what I see, they simply duplicate non-local files.

Reply:
ok